src/xcode/ENA/ENA/Source/Scenes/ExposureDetection/ExposureDetectionViewModel.swift

```diff
@@ -43,7 +43,7 @@ class ExposureDetectionViewModel: CountdownTimerDelegate {
 				switch $0 {
 				case .downloading:
 					self?.setupForDownloadingState()
-				case .detecting:
+				case .detecting, .riskRequested:
```

Comment:
In my opinion, this change will lead to a wrong behavior at the activity state UI. It will lead to showing the detection state immediately. But the correct process is to show the downloading state and the detecting state only if the detection has started.

Reply:
I see the same issue as Artur here, you can see the risk card jumping every time the home tab is opened even though we don't actually calculate the risk every time.

Reply:
Update: Me and Arthur had a small discussion together with Anna-Maria and I have incorporated those changes.